lib/enrollment.rb

```diff
@@ -1,35 +1,10 @@
-require 'pry'
-require_relative 'district'
 class Enrollment
-  attr_accessor :name, :school_data #:high_school_graduation_rates,
+  attr_accessor :name, :high_school_graduation_rates
+  attr_reader :kindergarten_participation
 
-  def initialize(enrollment_data)
-    @name = enrollment_data[:name]
-
-    @school_data = {kindergarten_participation: enrollment_data[:kindergarten_participation], high_school_graduation_rates: enrollment_data[:high_school_graduation_rates]}
-  end
-
-  def kindergarten_participation
```

Comment:
Once again, we're turning these explicit getter/setter methods into attr_accessors/readers.